src/Elastic.Markdown/Slices/Layout/_LandingPage.cshtml

```diff
@@ -167,8 +167,8 @@
 					</div>
 				</div>
 				<div class="flex flex-col rounded-xl bg-white">
-					<h3 class="font-sans font-bold text-ink-dark text-2xl">Query Languages</h3>
-					<p class="grow mt-3">Browse the docs for Query DSL, ES|QL, KQL, and more.</p>
+					<h3 class="font-sans font-bold text-ink-dark text-2xl">Query languages</h3>
+					<p class="grow mt-3">Browse the docs for Query DSL, <nobr>ES|QL</nobr>, KQL, and more.</p>
```

Comment:
Per https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/nobr

> Warning: Although this element is widely supported, it was never standard HTML, so you shouldn't use it. Instead, use the CSS property [white-space](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space) like this:

Reply:
Woops. Sorry for the bad suggestion 😅

Reply:
In that case we should use 

```html
<span class="whitespace-nowrap">ES|QL</span>
```

Reply:
Corrected